app/Alexa/Models/Role.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Alexa\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Role extends Model
```

Comment:
I can foresee that we will need to check if the role is admin or user.

Therefore, let's make it easier for us, if you agree:

- Add constant properties in class to map the enum values.
```php
    /**
     * Administrator role type.
     *
     * @var string
     */
    const ROLE_ADMINISTRATOR = 'admin';
```

- Add methods `isAdmin` and `isUser` to avoid making string comparisons outside the class.
```php
    /**
     * Check if this entity role is of admin type.
     *
     * @return bool
     */
    public function isAdmin()
    {
        return $this->type === self::ROLE_ADMINISTRATOR;
    }
```

- Add tests for these two methods (call me if you need help)
   - You're going to deal with [model factories](https://laravel.com/docs/5.5/database-testing#writing-factories) and [testing](https://laravel.com/docs/5.5/testing) conventions.